src/utils/search.js

```diff
@@ -30,22 +30,21 @@ export let transformNodeFromField = args => node => ({
   ...newNodeFromField(args),
 })
 
-export let indent = (Tree, parent, node, skipDefaultNode) => {
+export let indent = async (Tree, parent, node, skipDefaultNode) => {
```

Comment:
I'm not sure this needs to be async - see below.

Also as @re1ro and I discussed, this seems like something we should consider moving into `contexture-client`. The only reason it's here was because of the `blankNode` stuff, but that seems like the canonical thing to do when indenting anyway. The client has generally been ignorant of schemas, but that causes stuff like this to be orphaned and live in contexture-react instead.

Reply:
See my comment above for the async stuff.

> The client has generally been ignorant of schemas, but that causes stuff like this to be orphaned and live in contexture-react instead.

This function is not aware of schemas though.